src/components/modal/Modal.tsx

```diff
@@ -28,18 +28,15 @@ export const Modal = ({
   onClose,
   large,
 }: ModalProps) => {
-  const refDiv = useRef(null);
-
   return (
     <>
       <Transition appear show={isOpen} as={Fragment}>
         <Dialog
-          initialFocus={refDiv}
```

Comment:
focusing on the button adds a circle around it, need to fix

Reply:
done